package.json

```diff
@@ -161,9 +161,11 @@
     "wait-for-expect": "^1.1.1",
```

Comment:
Did you mean to update these dependencies? If so, how come?

Reply:
I did not and am unsure of why this would be necessary.